Source/loadsave.cpp

```diff
@@ -1528,6 +1530,45 @@ void SaveDroppedItemLocations(SaveHelper &file, const std::unordered_map<uint8_t
 	}
 }
 
+constexpr uint32_t VersionAdditionalMissiles = 0;
+
+void SaveAdditionalMissiles(int missileCountAdditional)

```

Comment:
What's the reason you went with only saving additional missiles to the expansion file?

Reply:
Cause this simplifies loading. 😉 
We don't need to check if the new missiles are present and ignore the original missiles if they exists.

Also we don't write the same content twice.

Reply:
I was going to just overwrite the content loaded from the original file :)